src/osd/PrimaryLogPG.cc

```diff
@@ -726,7 +726,7 @@ void PrimaryLogPG::maybe_force_recovery()
     return;
 
   // find the oldest missing object
-  version_t min_version = 0;
+  version_t min_version = pg_log.get_log().head.version;
```

Comment:
could you elaborate a little bit the rationale of this change in the commit message?

Reply:
if min_version is 0, if (min_version > xxx) always return false

Reply:
the question is why `pg_log.get_log().head.version`  is correct, but `0` is not.

Reply:
what we want to get is the minimum of the objects' versions. so we should initialize to as head version and compare it with other version

Reply:
oic, could you amend the commit message with the explanation above?